# std::vector<float64> end_jnts = {0,0,0,0}
float64 landing_leg_1_cmd
float64 landing_leg_2_cmd
float64 landing_leg_3_cmd
float64 landing_leg_4_cmd
